blockchain/contracts/Dice.sol

```diff
@@ -0,0 +1,32 @@
+//Contract based on [https://docs.openzeppelin.com/contracts/3.x/erc721](https://docs.openzeppelin.com/contracts/3.x/erc721)
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "hardhat/console.sol";
+
+contract TabletopDiceNFT is ERC721URIStorage, Ownable {
```

Comment:
I'm accustom to using ERC721Enumerable so that marketplaces like OpenSea can scan and import our NFTs automatically once they know the contract address. I attempted to implement it in a PR to your PR but things changed a little more than I anticipated so I didn't go through with it.